slurm.spec

```diff
@@ -594,16 +605,16 @@ rm -rf %{buildroot}
 
 %files torque
 %defattr(-,root,root)
-%{_bindir}/pbsnodes
```

Comment:
now from build script

Reply:
que?

Reply:
the build script strips these, no need for a patch in the spec file